content/docs/0.8.1/concepts.md

```diff
@@ -73,4 +87,31 @@ If the current healthy replica count is more than specified replica count, Longh
 ### Volume parameters
 
 #### Stale Replica Timeout 
-Stale Replica Timeout (`staleReplicaTimeout`) determines when would Longhorn cleanup an error replica after the replica become `ERROR`. Unit is in minutes. Default is `2880` (48 hours)
\ No newline at end of file
+Stale Replica Timeout (`staleReplicaTimeout`) determines when would Longhorn cleanup an error replica after the replica become `ERROR`. Unit is in minutes. Default is `2880` (48 hours)
+
+### Disaster Recovery Volumes
+
+A disaster recovery volume (DRV) is a special volume that stores data in a backup cluster in case the whole main cluster goes down. DRVs are used to increase the resiliency of Longhorn volumes.
```

Comment:
I would prefer to replace `DRV` with `DR volume`. Not sure when did we start calling DR volumes DRV...

Reply:
replaced throughout